db/sql/dbsetup.sql

```diff
@@ -359,6 +380,10 @@ RIGHT JOIN fhir_endpoints AS endpts ON links.url = endpts.url
 LEFT JOIN fhir_endpoints_info AS endpts_info ON endpts.url = endpts_info.url   
 LEFT JOIN vendors ON endpts_info.vendor_id = vendors.id
 LEFT JOIN npi_organizations AS orgs ON links.organization_npi_id = orgs.npi_id
+LEFT JOIN (SELECT fom.id as id, array_agg(fo.organization_name) as organization_names 
```

Comment:
Since these 4 lines (383-386) are repeated in this file twice and at least one other place (archivefile), I'm wondering if it could be made into a separate view, and then that view could be used elsewhere? i'm not sure if you can call a view in a view, but I feel like it would improve efficiency if this complicated query only had to be done once in a separate view.

Reply:
Also on the topic of efficiency, it might be worth adding indices for the new reference ids? I'm not entirely sure how indices work, but it looks like we have them for most ids that are used across tables. We could also add them for the validations table too since i'm not seeing those either